pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "bulk-data-service"
-version = "0.2.3"
+version = "0.2.5"
```

Comment:
Should this be `0.2.4`?

Reply:
There was an intermediate 0.2.4 that was released to dev, but not production, that's why there's a jump.

Reply:
Ah that makes sense, all good otherwise :)